shared/misc/init_container.sh

```diff
@@ -37,12 +40,22 @@ then
     export CATALINA_BASE=
 fi
 
-# If no app is published at /home/site/wwwroot/webapps, use the parking page app
+# If no app is published at /home/site/wwwroot/webapps, use the parking page app.
+# If an app is published, the default behavior is to copy the app to a local location, unless:
+# 1. WEBSITE_LOCAL_COPY_ENABLED is defined to 0 or FALSE, OR,
+# 2. Local cache is enabled, in which case making a local copy again is unnecessary.
 if [ ! -d /home/site/wwwroot/webapps ]
 then
-    cp -r /tmp/tomcat/webapps /usr/local/tomcat/webapps
+    SITEROOT=/tmp/tomcat/webapps
+    echo "Using parking page app with SITEROOT=$SITEROOT"
+elif [[ "$WEBSITE_LOCAL_CACHE_OPTION" = "Always" || "$WEBSITE_LOCAL_COPY_ENABLED" = "0"  || "$WEBSITE_LOCAL_COPY_ENABLED" = "false" ]]
```

Comment:
>EBSITE_LOCAL_COPY_ENABLED [](start = 55, length = 25)

The local copy should be by default and customers can opt out, so I think the app setting should be called WEBSITE_SKIP_LOCALCOPY=1 or something like that

Reply:
Changed WEBSITE_LOCAL_COPY_ENABLED to WEBSITE_DISABLE_LOCAL_COPY.